tests/cases/onboarders_care_platform/case_onboarders_participants.yaml

```diff
@@ -190,4 +190,27 @@ TestOnboarderClickandViewParticipantRecordfromListing:
       Asserts:
         - Type: contain
           Var: SUBTAB_CLASS
-          Value: 'active'
\ No newline at end of file
+          Value: 'active'
+
+# CXTB-146 Participant Profile - Review, Edit or Add Vendor Info
+TestOnboarderCarePartnerParticipantReviewEditOrAddVendorInfo:
```

Comment:
please add the "Profile" word to the test name, like "TestOnboarderCarePartnerParticipantProfileReviewEditOrAddVendorInfo"